CONTRIBUTING.md

```diff
@@ -1,6 +1,38 @@
 # Contributing to Profanity
 
-`make doublecheck` will run the code formatter, spell checker and unit tests.
+## Submitting patches
+We recommend for people to always work on a dedicated git branch for each fix or feature.
+Don't work on master.
+So that they can easily pull master and rebase their work if needed.
+
+For fixing (reported) bugs we usually use `git checkout -b fix/issuenumber-somedescription`.
+When working on a new feature we usually use `git checkout -b feature/optionalissuenumber-somedescription`.
+
+However this is not a rule just a recommendation to keep an overview of things.
+If your change isn't a bugfix or new feature you can also just use any branch name.
+
+### GitHub
+We would like to encourage people to use GitHub to create pull requests.
+It makes it easy for us to review the patches, track WIP branches, organize branches with labels and milestones,
+and help others to see what's being worked on.
+
+Also see the blogpost [Contributing a Patch via GitHub](https://profanity-im.github.io/blog/post/contributing-a-patch-via-github/).
+
+### E-Mail
+In case GitHub is down or you can't use it for any other reason, you can send a patch to your [mailing list](https://lists.notraces.net/mailman/listinfo/profanity).
```

Comment:
```suggestion
In case GitHub is down or you can't use it for any other reason, you can send a patch to our [mailing list](https://lists.notraces.net/mailman/listinfo/profanity).
```